tests/depth-dependent-3d-shell-list-simple.prm

```diff
@@ -0,0 +1,89 @@
+# A simple setup for convection in a 3d shell. See the
+# manual for more information.
```

Comment:
Please update the comment to show what it is you are testing here. Probably same for the other tests.